src/charm.py

```diff
@@ -27,16 +36,37 @@
 
 CACHE_PATH = "/var/lib/nginx/proxy/cache"
 CONTAINER_NAME = "content-cache"
+EXPORTER_CONTAINER_NAME = "nginx-prometheus-exporter"
 CONTAINER_PORT = 80
 REQUIRED_JUJU_CONFIGS = ["site", "backend"]
 
 
 class ContentCacheCharm(CharmBase):
-    """Charm the service."""
+    """Charm the service.
+
+    Attrs:
+        on: Ingress Charm Events
+        ERROR_LOG_PATH: NGINX error log
+        ACCESS_LOG_PATH: NGINX access log
+        _metrics_endpoint: Provider of metrics for Prometheus charm
```

Comment:
It is good to document these as well, I think the private attributes are optional just FYI